components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -577,6 +577,10 @@ service Policies {
 
   A project defines the scope of resources in a policy statement. Resources can be in more than one project.
 
+  When a project is created, the system also creates three policies associated with the new project, 
+  one for each of the following roles: editor, viewer, and project owner.
```

Comment:
either `editor, viewer, and owner.` or `project editor, project viewer, and project owner.`

Reply:
I'd rather keep the specific role names. The project owner policy uses `project owner` role, not `owner`